pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1704,7 +1704,7 @@ func (l *LibvirtDomainManager) DeleteVMI(vmi *v1.VirtualMachineInstance) error {
 	}
 	defer dom.Free()
 
-	err = dom.UndefineFlags(libvirt.DOMAIN_UNDEFINE_NVRAM)
+	err = dom.UndefineFlags(libvirt.DOMAIN_UNDEFINE_KEEP_NVRAM)
```

Comment:
Just to confirm, this is fine anyway (even if the VM does not have any backend storage) because we dispose of the empyDir anyway when we kill the VM/pod? IOW libvirt won't complain if you didn't explicitly define a file-backed NVRAM?